aws/aws-config.json

```diff
@@ -33,6 +32,6 @@
     },
 
     "docker": {
-        "image_path": "rahulsm2/honeybadgermpc:latest"
+        "image_path": "smkuls/honeybadgermpc:latest"
```

Comment:
Similarly if dockerhub is an essential part of our AWS usage we should document that specifically

Reply:
This has already been documented in the AWS Readme. I can create a HoneyBadgerMPC repo on Dockerhub and remove my username from this.

Reply:
I have created a `dsluiuc` organization on DockerHub. We can use that from now on.